apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -179,7 +182,15 @@ const Item = ({
   ) : (
     <Link
       href={`/event-types/${type.id}?tabName=setup`}
-      className="flex-1 overflow-hidden pr-4 text-sm"
+      className={cx("flex-1 overflow-hidden pr-4 text-sm", isReordering && "cursor-grabbing")}
+      onMouseDown={(e) => {
+        e.preventDefault();
+      }}
+      onClick={(e) => {
+        if (isReordering) {
+          e.preventDefault();
```

Comment:
When you drag and drop it at the same position, it counts it as a click and so navigates to the link. This is to prevent that